### ISSUE TITLE
Profile photo and granularity in cronjobs

### ISSUE THREAD

**Issue by gianfelicevincenzo** [sct/overseerr#3870]
### Description

Allow to upload your profile photo and be more precise regarding the cronjob time settings. Preset values ​​are not right for everyone. I would like the radar scan, as well as the plex library scan, to be done faster

### Desired Behavior

1 - Upload profile photo
2 - Cron job time granularity

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by ajw107** [comment]
Love the idea of the Profile Photos.  The cron job thing, I assume you mean you want to be able the change the frequency of the jobs, not their speed (that's hardware/library size dependent, surely)?

**Comment by gianfelicevincenzo** [comment]
Yes, i mean to be able to change the frequency of cronjobs. Sorry for my bad english
